pylint/message/message_handler_mix_in.py

```diff
@@ -347,18 +348,29 @@ def add_one_message(
             )
         )
 
-    def print_full_documentation(self, stream=None):
-        """output a full documentation in ReST format"""
-        if not stream:
-            stream = sys.stdout
+    def _get_checkers_infos(self) -> dict:
+        by_checker: dict = {}
```

Comment:
I'm not sure we can use type assignments here, given we still need to run on 3.4 and 3.5.